src/resources/views/crud/fields/upload_multiple.blade.php

```diff
@@ -40,7 +40,7 @@
 		<input
 	        type="file"
 	        name="{{ $field['name'] }}[]"
-	        value="@if (old(square_brackets_to_dots($field['name']))) old(square_brackets_to_dots($field['name'])) @elseif (isset($field['default'])) $field['default'] @endif"
+	        value="@if (old(square_brackets_to_dots($field['name']))) {{ old(square_brackets_to_dots($field['name'])) }} @elseif (isset($field['default'])) {{ $field['default'] }} @endif"
```

Comment:
Hum .. how to say it .. No! 🙃 

We can't @promatik , actually there is no sense at all to have `value=smt` in that input as it is totally impossible to set a `file` type input value programatically. 

There are only 2 ways of dealing with this: 
- Validate the form with ajax and only submit when you are sure form is not going to fail, so ... yeah.
- Before validation check if file exists in request, save as temporary and setup a mechanism to use that file if user didn't change it, otherwise do the cleanup of unused files. 

Either way .. I just remove this `value=` from here, and let's hope for dropzone 🤞 Or forms validating with ajax before submiting. (that would be neat!)

Reply:
I have no comments regarding this. There is nothing for us to do from my point of view, that's how upload inputs work, they _can't_ prepopulate with the old value.